setup.py

```diff
@@ -10,17 +10,17 @@ def read(fname):
 
 
 base_packages = [
-    "scikit-learn>=0.24.0",
-    "pandas>=1.1",
-    "matplotlib>=3.3.4",
-    "joblib>=1.0.1",
-    "tqdm>=4.59.0",
-    "shap>=0.39.0",
-    "numpy>=1.20.1",
+    "scikit-learn==0.24.0",
```

Comment:
Same comment as before, do we want to pin specific version of library as requirement here?
This is going to possibly cause that probatus is less usable by the users, that want to use e.g. different version of sklearn. If we don't do that there is a possibility that some of the packages updates and our tests break suddenly. However, I think if we want to make sure the most users can use the package, maybe it is better to leave it as it was.
What do you think about this?

Reply:
No, I don't want to pin. You're not looking at the latest version: https://github.com/ing-bank/probatus/blob/96-dependabot/setup.py